### ISSUE TITLE
AttributeError: 'CompilationUnit' object has no attribute 'filename_of_contract'

### ISSUE THREAD

**Issue by Jovonni** [trailofbits/manticore#2519]
### Summary of the problem
When running the simple python file using python ```3.7.0``` & ```3.7.6``` & ```3.6.5``` & ```3.9.9```:

```python
from manticore.ethereum import ManticoreEVM

m = ManticoreEVM()
print(m)

user_account = m.create_account(balance=1000)
print(user_account)

source_code = '''
pragma solidity >=0.7.3 <=0.8.6;
contract Simple {
    function f(uint a) payable public{
        if (a == 65) {
            revert();
        }
    }
}
'''

print(source_code)

# Initiate the contract
contract_account = m.solidity_create_contract(source_code, owner=user_account)
```

## Error
```bash
2022-01-08 22:30:04,785: [30576] m.c.manticore:WARNING: Manticore is only supported on Linux. Proceed at your own risk!
<manticore.ethereum.manticore.ManticoreEVM| Alive States: 1; Running States: 0 Terminated States: 0 Killed States: 0 Started: False Killed: False>
907623960778932583730695191281088592966447129015

pragma solidity >=0.7.3 <=0.8.6;
contract Simple {
    function f(uint a) payable public{
        if (a == 65) {
            revert();
        }
    }
}

Traceback (most recent call last):
  File "manticore_test.py", line 23, in <module>
    contract_account = m.solidity_create_contract(source_code, owner=user_account)
  File "/Users/jovi/.pyenv/versions/3.7.6/lib/python3.7/site-packages/manticore/ethereum/manticore.py", line 576, in solidity_create_contract
    source_code, contract_name_i, libraries=deps, crytic_compile_args=compile_args
  File "/Users/jovi/.pyenv/versions/3.7.6/lib/python3.7/site-packages/manticore/ethereum/manticore.py", line 346, in _compile
    temp.name, contract_name, libraries, crytic_compile_args
  File "/Users/jovi/.pyenv/versions/3.7.6/lib/python3.7/site-packages/manticore/ethereum/manticore.py", line 305, in _compile_through_crytic_compile
    filename = compilation_unit.filename_of_contract(name).absolute
AttributeError: 'CompilationUnit' object has no attribute 'filename_of_contract'
```

### Manticore version

```bash
0.3.6 #probably ran other versions too as I was trying multiple versions of python
```

### Python version

```bash
3.7.0
3.7.6
3.6.5
3.9.9
```

### OS / Environment

MacOS 10.15.7

### Dependencies

```
crytic-compile==0.2.2
cytoolz==0.11.2
eth-hash==0.3.2
eth-typing==3.0.0
eth-utils==2.0.0
future==0.18.2
importlib-metadata==4.10.0
intervaltree==3.1.0
manticore==0.3.6
ply==3.11
prettytable==3.0.0
protobuf==3.19.1
pyevmasm==0.2.3
pysha3==1.0.2
PyYAML==6.0
rlp==3.0.0
sortedcontainers==2.4.0
toolz==0.11.2
typing-extensions==4.0.1
wasm==1.2
wcwidth==0.2.5
z3-solver==4.8.14.0
zipp==3.7.0
```

### Step to reproduce the behavior
Run the python basic file above

### Expected behavior
To successfully run the example python script

### Actual behavior
Throw the error shown above

**Comment by rickbeeloo** [comment]
Same here

**Comment by rickbeeloo** [comment]
Doesn't really look like soemone is actively handling issues on the github tho :(

**Comment by rickbeeloo** [comment]
@Jovonni can fix this with `pip install crytic-compile==0.2.1`
